src/components/DiceGame/Dice.js

```diff
@@ -0,0 +1,461 @@
+                                            <span>Дата:</span>
+                                            <span>
+                                                {historyItem.dateTime}
+                                            </span>{' '}
+                                        </div>
+                                        <div>
+                                            <span>Ставка:</span>
+                                            <span>
+                                                {' '}
+                                                {getBetValue(
+                                                    historyItem.betType
+                                                )}
+                                            </span>
+                                        </div>
+                                        <div>
+                                            <span>Результат:</span>{' '}
+                                            <span>
+                                                {' '}
+                                                {historyItem.droppedNumber}
+                                            </span>
+                                        </div>
+                                        <div>
+                                            <span>
+                                                Ставка
+                                                {historyItem.isBetWon
+                                                    ? ''
+                                                    : ' не'}{' '}
+                                                сыграла:
+                                            </span>
+                                            <span>
+                                                {' '}
+                                                {historyItem.isBetWon
+                                                    ? '✅'
+                                                    : '❌'}
+                                            </span>
+                                        </div>
+                                        <hr />
+                                    </li>
+                                )
+                            })}
+                        </ol>{' '}
+                    </div>
+                )}
+            </div>
+            {isLoginModalVisible && (
+                <LoginModal
+                    setToken={setToken}
+                    setIsLoginModalVisible={setIsLoginModalVisible}
+                    setIsRegistrationModalVisible={
+                        setIsRegistrationModalVisible
+                    }
+                />
+            )}
+            {isRegistrationModalVisible && (
+                <RegistrationModal
+                    setToken={setToken}
+                    setIsRegistrationModalVisible={
+                        setIsRegistrationModalVisible
+                    }
+                />
+            )}
+            {showPopUp && (
```

Comment:
Would make this pop up centered and make background darker/muted when the pop up is shown.

Reply:
done